app/importers/file_importers/csv_downloader.rb

```diff
@@ -10,9 +10,9 @@ def initialize(log:, client:, remote_file_name:, transformer:)
   end
 
   def get_data
-    file = download_file
+    file_contents = download_file
 
-    data = @transformer.transform(file)
+    data = @transformer.transform(file_contents)
```

Comment:
Clarifying that the previous API to `#transform` was to pass the bytes of the CSV file as as string, not a file handle.

Reply:
Yeah, good name change. 👍